honeybee_radiance/sensorgrid.py

```diff
@@ -190,6 +198,40 @@ def count(self):
         """Number of sensors."""
         return len(self._sensors)
 
+    @property
+    def room_identifier(self):
+        """Get or set text for the Room identifier to which this SensorGrid belongs.
+        
+        This will be used in the model_folder_dict method to narrow down the
+        number of aperture groups that have to be run with this sensor grid.
+        If None, the grid will be run with all aperture groups in the model.
+        """
+        return self._room_identifier
+
+    @room_identifier.setter
+    def room_identifier(self, n):
+        self._room_identifier = typing.valid_string(n)
+
+    def model_folder_dict(self, model=None):
+        """Get a dictionary to be written into the folder structure.
+
+        When written into a JSON, this dictionary will narrow down the number
+        of aperture groups that have to be run with this sensor grid.
+
+        Args:
+            model: A honeybee Model object which will be used to identify
+                the aperture groups that will be run with this sensor grid.
+        """
+        base = {'count': self.count}
+        if model:  # add the aperture_groups key
+            if self._room_identifier:  # add only the groups in the light path
+                base['aperture_groups'] = \
+                    light_path_from_room(model, self._room_identifier)
+            else:  # add all groups across the entire model
+                base['aperture_groups'] = \
+                    [[grp_id] for grp_id in model.subface_group_identifiers]
```

Comment:
This is probably not required. If the model is not provided the ray-tracing will happen for all the glass resources anyways.

Reply:
Ok. I will cut it out, then. This should save some time in the export process.